lib/portage/repository/config.py

```diff
@@ -74,29 +74,60 @@ def _find_invalid_path_char(path, pos=0, endpos=None):
 class RepoConfig(object):
 	"""Stores config of one repository"""
 
-	__slots__ = ('aliases', 'allow_missing_manifest', 'allow_provide_virtual',
-		'auto_sync', 'cache_formats', 'clone_depth',
-		'create_manifest', 'disable_manifest',
-		'eapi', 'eclass_db', 'eclass_locations', 'eclass_overrides',
-		'find_invalid_path_char', 'force', 'format', 'local_config', 'location',
-		'main_repo', 'manifest_hashes', 'masters', 'missing_repo_name',
-		'name', 'portage1_profiles', 'portage1_profiles_compat', 'priority',
-		'profile_formats', 'sign_commit', 'sign_manifest', 'strict_misc_digests',
-		'sync_depth', 'sync_hooks_only_on_change',
-		'sync_type', 'sync_umask', 'sync_uri', 'sync_user', 'thin_manifest',
-		'update_changelog', 'user_location', '_eapis_banned', '_eapis_deprecated',
-		'_masters_orig', 'module_specific_options', 'manifest_required_hashes',
+	__slots__ = ('aliases',
+		'allow_missing_manifest',
+		'allow_provide_virtual',
```

Comment:
if your going to use indent, then do them all indented, finish with the ) on it's own line less one indent.

foo = (
    "one",
    "two",
    ...
    "five",
)

then it is easier to add/remove individual entries without causing a diff on other entries due to a moving of "one" or the end ")"   They are more uniform overall.

Reply:
damn comments remove space indents....

Reply:
Done, thanks!.